README.md

```diff
@@ -1,2 +1,12 @@
-# template-dbus-service
-Template for creating a D-Bus service component
+# Template D-Bus service
+
+Template for creating a D-Bus service component.
+
+
+# License and Copyright
+
+Copyright (C) 2019 Luxoft Sweden AB
```

Comment:
&copy;

Reply:
We seem to mix in our repos. But OK, will change.